How-To/Use-the-Data-team-server.md

```diff
@@ -0,0 +1,63 @@
+# How to use the Data team server
+
+## Contents
+
+* [What is the Data team server?](#what-is-the-data-team-server)
+* [Who controls the server?](#who-controls-the-server)
+* [What lives on the server?](#what-lives-on-the-server)
+* [How do I access the server?](#how-do-i-access-the-server)
+* [How do I add or remove users from the server?](#how-do-i-add-or-remove-users-from-the-server)
+
+## What is the Data team server?
+
+The Data team server is a remote machine that we use to host various tools and services. It is an AWS EC2 instance running Ubuntu in the `us-east-1` region.
+
+A sample of the types of things we use the server for include:
+
+* Run RStudio and JupyterLab instances with more compute than we have on our laptops 
+* Schedule cron jobs to perform ETL tasks
+* Run web apps using Docker
+
+These examples are intended to give you a high-level overview of the types of tools we host on the server. For more detail on the tools that are available on the server, see: [What's running on the server?](#what-is-running-on-the-server)
+
+## Who controls the server?
+
+The server is owned and maintained collectively by the Data team staff, but only a small group of Senior Data Scientists have superuser permissions (these users are `sudoers` in Linux terminology). If you need access to superuser permissions (to run commands that require `sudo`, for example), contact a Senior Data Scientist.
+
+## What lives on the server?
+
+The following is a list of tools and services that live on the server as of summer 2023:
+
+| Path | Description | Repo | Run by |
+| ---- | ----------- | ---- | ------ |
+| `/home/<username>` | Home directories for each member of the data team | | |
+| `/home/shiny-server/services` | Home directory for various apps that run on the server | | |
+| `/home/shiny-server/services/service-sqoop-iasworld` | Service that extracts data from the iasWorld platform and loads it into AWS S3 | [`service-sqoop-iasworld`](https://github.com/ccao-data/service-sqoop-iasworld) | cron |
+| `/home/shiny-server/services/service-socrata-ingress-agent` | Service that extracts data from AWS Athena and loads it into the Socrata data portal | [`service-socrata-ingress-agent`](https://github.com/ccao-data/service-socrata-ingress-agent) | systemd |
+| `/home/shiny-server/services/service-watchtower` | Service that keeps Docker images up to date on the server | [`service-watchtower`](https://github.com/ccao-data/service-watchtower) | docker |
```

Comment:
Are these older Docker services run via direct invocations of Docker, or via the `service-shiny-proxy` app?

Reply:
The apps are all launched via `service-shiny-proxy`, everything else is run via `docker-compose`.